.github/ISSUE_TEMPLATE/COMMUNITY_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!-- (REQUIRED) What is the nature of this issue? -->
+## This Community doc request is a:
+- [ ] New topic request
+- [ ] Topic content updates
+- [ ] Other
+
+<!-- (REQUIRED) What is the Community issue and code PR? -->
+## Community GitHub issue link:
+
+## Community GitHub code PR link:
+
+<!-- (REQUIRED) What is the Magento Open Source/Commerce and version for the PR? Will it be back or forward ported? -->
```

Comment:
If we stick with this format, all comments should appear below the heading of the section to which they pertain throughout the template. For example:

```
## Magento version:
<!-- (REQUIRED) What is the Magento Open Source/Commerce and version for the PR? Will it be back or forward ported? -->
- [ ] Open Source
- [ ] Commerce
```

Reply:
Yeah I like your format suggestions. We may have other info to check off and add. Will tweak into the table idea.